gradio/components.py

```diff
@@ -26,11 +26,11 @@
 import pandas as pd
 import PIL
 import PIL.ImageOps
-from PIL import Image as _Image  # using _ to minimize namespace pollution
 from ffmpy import FFmpeg
 from markdown_it import MarkdownIt
 from mdit_py_plugins.dollarmath import dollarmath_plugin
 from pandas.api.types import is_numeric_dtype
+from PIL import Image as _Image  # using _ to minimize namespace pollution
```

Comment:
Ok I was getting this locally too when I linted the backend. 

I'm guessing the person who added this line had a different version of `isort`?